shop/src/main/java/ro/msg/learning/shop/repository/OrderDetailRepository.java

```diff
@@ -0,0 +1,10 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.domain.OrderDetail;
+
+import java.util.UUID;
+@Repository
+public interface OrderDetailRepository extends JpaRepository<OrderDetail, UUID> {
```

Comment:
Id of the entity is OrderDetailId, so it should be public interface OrderDetailRepository extends JpaRepository<OrderDetail, OrderDetailId>